spec/support/blueprints.rb

```diff
@@ -177,6 +177,7 @@ def first_or_make_site_unless_manufacturer(institution)
   specimen_role { "p" }
   date_produced { Faker::Time.backward }
   measured_signal { 10.0 }
+  batch { object.encounter.try(:batch) || Batch.make! }
```

Comment:
This is wrong: a sample doesn't need to come from a batch. You should manually associate the sample to a batch in your tests (or use the `:batch` trait below). For example:

`Sample.make!(:filled, batch: Batch.make)`